mmcv/ops/csrc/common/cuda/convex_iou_cuda_kernel.cuh

```diff
@@ -402,28 +432,9 @@ __device__ inline void Jarvis(Point* in_poly, int& n_poly) {
   double sign;
   Point right_point[10], left_point[10];
 
-  for (int i = 0; i < n_poly; i++) {
-    if (in_poly[i].y < in_poly[0].y ||
-        in_poly[i].y == in_poly[0].y && in_poly[i].x < in_poly[0].x) {
-      Point* j = &(in_poly[0]);
-      Point* k = &(in_poly[i]);
-      swap1(j, k);
-    }
-    if (i == 0) {
-      p_max = in_poly[0];
-      max_index = 0;
-    }
-    if (in_poly[i].y > p_max.y ||
-        in_poly[i].y == p_max.y && in_poly[i].x > p_max.x) {
-      p_max = in_poly[i];
-      max_index = i;
-    }
-  }
-
-  if (max_index == 0) {
-    max_index = 1;
-    p_max = in_poly[max_index];
-  }
+  quickSort(in_poly, n_poly);
+  max_index = n_poly - 1;
```

Comment:
The for loop inside the while loop below begin with `int i = 1`, `in_poly[0]` will never been visited. Is it expected?

Reply:
`in_poly[0]` is the most top-left point, serving as the starting point in `Jarvis`, which is visited by the index `top1` and `top2`.